docusaurus/docs/guides/deployments/10-linux/20-router/30-configuration.md

```diff
@@ -0,0 +1,228 @@
+      options:
+        outQueueSize:   4
+
+listeners:
+# bindings of edge and tunnel requires an "edge" section below
+  - binding: edge
+    address: tls:0.0.0.0:3022
+    options:
+      advertise: router1.ziti.example.org:3022
+      connectTimeoutMs: 5000
+      getSessionTimeout: 60
+
+
+
+edge:
+  csr:
+    country: US
+    province: NC
+    locality: Charlotte
+    organization: NetFoundry
+    organizationalUnit: Ziti
+    sans:
+      dns:
+        - localhost
+        - router1.ziti.example.org
+      ip:
+        - "127.0.0.1"
+
+forwarder:
+  latencyProbeInterval: 0
+  xgressDialQueueLength: 1000
+  xgressDialWorkerCount: 128
+  linkDialQueueLength: 1000
+  linkDialWorkerCount: 32
+```
+
+</TabItem>
+
+<TabItem value="Gateway" label="Private Router with Tunneler">
+
+This is an example of generating a private router configuration with the `ziti` CLI. 
+
+<br />
+
+```text
+#!/usr/bin/env bash
+
+# working directory for the router
+export ZITI_HOME=/var/lib/ziti-router
+
+# address and port of control plane endpoint
+export  ZITI_CTRL_ADVERTISED_ADDRESS=ctrl.ziti.example.org \
+        ZITI_CTRL_ADVERTISED_PORT=1280
+
+# address and port of this router
+export  ZITI_ROUTER_ADVERTISED_ADDRESS=router2.ziti.example.org \
+        ZITI_ROUTER_PORT=3022 \
+        ZITI_ROUTER_LISTENER_BIND_PORT=3022
+
+ziti create config router edge \
+    --routerName router2 \
+    --tunnelerMode tproxy \
+    --private
+```
+
+<br />
+
+Public routers dial and listen for other routers dialing mesh links. Private routers dial for mesh links but do not listen. The `edge` capability is independent of the `link` capability, and so both private and public routers may listen for and advertise edge connections from endpoints.
+
+<br />
+
+### Private Router with Tunneler config.yml
+
+```text
+v: 3
+
+identity:
+  cert:             "/var/lib/ziti-router/router2.cert"
+  server_cert:      "/var/lib/ziti-router/router2.server.chain.cert"
+  key:              "/var/lib/ziti-router/router2.key"
+  ca:               "/var/lib/ziti-router/router2.cas"
+
+ctrl:
+  endpoint:             tls:ctrl.ziti.example.org:1280
+
+link:
+  dialers:
+    - binding: transport
+
+listeners:
+# bindings of edge and tunnel requires an "edge" section below
+  - binding: edge
+    address: tls:0.0.0.0:3022
+    options:
+      advertise: router2.ziti.example.org:3022
+      connectTimeoutMs: 5000
+      getSessionTimeout: 60
+  - binding: tunnel
+    options:
+      mode: tproxy
+      resolver: udp://127.0.0.1:53
+      lanIf: 
+
+edge:
+  csr:
+    country: US
+    province: NC
+    locality: Charlotte
+    organization: NetFoundry
+    organizationalUnit: Ziti
+    sans:
+      dns:
+        - localhost
+        - router2.ziti.example.org
+      ip:
+        - "127.0.0.1"
+
+
+forwarder:
+  latencyProbeInterval: 0
+  xgressDialQueueLength: 1000
+  xgressDialWorkerCount: 128
+  linkDialQueueLength: 1000
+  linkDialWorkerCount: 32
+
+```
+
+</TabItem>
+
+</Tabs>
```

Comment:
this whole file feels like it belongs merged into the `/docs/reference/configuration/router` page and some (much?) feels redundant?